src/apps/pulse/components/PreviewBuy.tsx

```
@@ -0,0 +1,221 @@
+              <Esc closePreview={props.closePreview} />
+            </div>
+          </div>
+        </div>
+
+        <div className="flex justify-between" style={{margin: 10, marginBottom: 0, fontSize: 13, color: "grey"}}>
+          <div>You’re paying</div>
+          <div className="flex">Total: ${totalPay}</div>
+        </div>
+
+        <div className="rounded-[10px]" style={{width: 422, minHeight: 50, background: "black", margin: 10}}>
+          {props.payingTokens.map((item) => <PayingToken payingToken={item}/>)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)